Week-3/day6/cards/styles.css

```diff
@@ -0,0 +1,75 @@
+.table {
+  width: 400px;
+  height: 300px;
+
+  /* Part of your solution goes here */
+  background-image: linear-gradient(to bottom, forestgreen, darkgreen);
+}
+
+.cardRed {
+  color: red;
+}
+
+.cardBlack {
+  color: black;
+}
+
+.card {
+  background-color: whitesmoke;
+  border: 1px solid black;         
+  width: 100px;
+  height: 160px;
+  line-height: 160px;
+  text-align: center;
+  font-size: 80px;
+
+  /* Part of your solution goes here */
+  position: releative;
```

Comment:
There's a typo in the property position: releative; it should be position: relative;.